src/Auth0Client.ts

```diff
@@ -92,8 +92,7 @@ const cacheFactory = (location: string) => {
 /**
  * @ignore
  */
-const supportWebWorker = () =>
-  !isIE11() && !isSafari10() && !isSafari11() && !isSafari12_0();
+const supportWebWorker = () => !isIE11();
```

Comment:
@frederikprijck is it fair to say we only added these Safari checks to stop it from using the web worker _purely_ because of `AbortConrtoller`, or were there other factors?

I haven't removed these functions, do you see value in keeping them around or should we remove them?

Reply:
It was to fix the AbortController: https://github.com/auth0/auth0-spa-js/issues/594

Reply:
If we revert this, we should test this again on Safari. LMK if you dont have access to BrowserStack, I dont mind testing it.

Reply:
Yeah I've tested, versions I tested are in the PR